metrix-mapping/src/main/java/com/powsybl/metrix/mapping/log/LimitLogBuilder.java

```diff
@@ -0,0 +1,66 @@
+package com.powsybl.metrix.mapping.log;
+
+import java.util.Objects;
+
+public class LimitLogBuilder extends AbstractLogBuilder implements LogDescriptionBuilder {
```

Comment:
QCA : RMQ => A builder should be static in the Builder pattern

Reply:
GEL: It's not mandatory